README.md

```diff
@@ -26,18 +26,6 @@ Write changes? [y] y
 Wrote "dune-project"
 ```
 
-It works as follows:
-
-1. Lists the `*.opam` files in your project's root (ensuring they're up-to-date, if generated).
-2. Runs `dune external-lib-deps --only-packages $PKG --unstable-by-dir @install` and `... @runtest` to get each package's ocamlfind dependencies.
-3. Filters out local dependencies using `dune describe` (for now; would be good to lint these too in future, but needs a different code path).
-4. Filters out vendored dependencies (by ignoring dependencies from subdirectories with their own `dune-project` file).
-5. For each ocamlfind library, it finds the corresponding opam library by
-   finding its directory and then finding the `*.changes` file saying which
-   opam package added its `META` file.
-6. Checks that each required opam package is listed in the opam file.
-7. For any missing packages, it offers to add a suitable dependency, using the installed package's version as the default lower-bound.
-
```

Comment:
If this is no longer the case, can it be replaced with an explanation of how it is working now?